mods/_modpack.dm

```diff
@@ -1,16 +1,30 @@
 /decl/modpack
-	var/name                         // A string name for the modpack. Used for looking up other modpacks in init.
+	/// A string name for the modpack. Used for looking up other modpacks in init.
+	var/name
+	/// A string desc for the modpack. Can be used for modpack verb list as description.
+	var/desc
+	/// A string author for the modpack.
+	var/author
 
-	var/list/dreams                  // A list of strings to be added to the random dream proc.
+	/// A list of strings to be added to the random dream proc.
+	var/list/dreams
 
-	var/list/credits_other           // A list of strings that are used by the end of round credits roll.
-	var/list/credits_adventure_names // As above.
-	var/list/credits_crew_names      // As above.
-	var/list/credits_holidays        // As above.
-	var/list/credits_adjectives      // As above.
-	var/list/credits_crew_outcomes   // As above.
-	var/list/credits_topics          // As above.
-	var/list/credits_nouns           // As above.
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_other
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_adventure_names
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_crew_names
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_holidays
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_adjectives
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_crew_outcomes
+	/// A list of strings that are used by the end of round credits roll.
+	var/list/credits_topics
+	/// A list of strings that are used by the end of round credits roll.
```

Comment:
Why make the comments worse? :(

Reply:
I did it to respect dmdoc :^(

Reply:
As Lohi said you can use `// !` instead of making it untidy.